README.md

```diff
@@ -31,13 +39,61 @@ export JWT_ISSUER=http://localhost:8082/
 export JWT_AUDIENCE=http://localhost:8082/
 export JWT_REALM=jehlomat_local_realm
 export SUPER_ADMIN_EMAIL=jehlomat@cesko.digital
-export MAILJET_PUBLIC_KEY=public-key-for-mailjet
-export MAILJET_PRIVATE_KEY=private-key-for-mailjet
+export MAILJET_PUBLIC_KEY=cbb81504a06f2fd735db577f09666b7f
+export MAILJET_PRIVATE_KEY=fdd479b47c4af6e4d46bb07bf889432d
 ```
-6) Run application
+6) cd service-jehlomat 
+7) ../gradlew shadowJar 
+8) java -jar build/libs/service-jehlomat-all.jar -config=./src/test/resources/application.conf
+
+## Run hybrid backend application with Docker
+
+> This approach is useful mostly for FE development.
+By hybrid means, the BE service will be run locally but the database will be run via docker, so it will not be necessary to install the DB and run it 1. locally.
+
+1. In the root run `docker compose up`
+2. Create relations:
+    1. Run `docker ps`, you should see a table with just one row. It is your running container. Remember its container id, we will need it in the next step
+    2. Run commands:
+    ```shell
+    docker exec -it 1f5a39c5e45d psql postgresql://jehlomat:jehlomat@localhost:5432/jehlomat -f ./service-jehlomat/src/main/resources/sql/postgis.sql
+    docker exec -it 1f5a39c5e45d psql postgresql://jehlomat:jehlomat@localhost:5432/jehlomat -f ./service-jehlomat/src/test/resources/obce.sql
+    docker exec -it 1f5a39c5e45d psql postgresql://jehlomat:jehlomat@localhost:5432/jehlomat -f ./service-jehlomat/src/test/resources/mc.sql
+    docker exec -it 1f5a39c5e45d psql postgresql://jehlomat:jehlomat@localhost:5432/jehlomat -f ./service-jehlomat/src/test/resources/okres.sql
+    docker exec -it 1f5a39c5e45d psql postgresql://jehlomat:jehlomat@localhost:5432/jehlomat -f ./service-jehlomat/src/main/resources/sql/create_table.sql
+    docker exec -it 1f5a39c5e45d psql postgresql://jehlomat:jehlomat@localhost:5432/jehlomat -f ./service-jehlomat/src/main/resources/sql/insert_super_admin.sql
+    ```
+3. Set env variables
 ```shell
-./gradlew run --parallel
+export DATABASE_HOST=localhost
+export DATABASE_PORT=5432
+export DATABASE_USERNAME=jehlomat
+export PGPASSWORD=jehlomat
+export DATABASE_NAME=jehlomat
+export JWT_ISSUER=http://localhost:8082/
+export JWT_AUDIENCE=http://localhost:8082/
+export JWT_REALM=jehlomat_local_realm
+export SUPER_ADMIN_EMAIL=jehlomat@cesko.digital
+export MAILJET_PUBLIC_KEY=cbb81504a06f2fd735db577f09666b7f
+export MAILJET_PRIVATE_KEY=fdd479b47c4af6e4d46bb07bf889432d
 ```
+6. cd service-jehlomat
+7. ../gradlew shadowJar
```

Comment:
Please, what does this command do actually?

Reply:
Please see: https://github.com/cesko-digital/jehlomat/pull/163/commits/e2e2199f31ec79331afafc74a63ccbb9afac5a2d